web-test-runner.config.js

```diff
@@ -50,6 +51,7 @@ export default {
   testRunnerHtml: (testFramework) => `
     <html>
       <head>
+        <meta name="testEnvironment" ${isDebugMode ? 'debug' : ''}>
```

Comment:
Initially, the idea was to skip a11y snapshot tests on puppeteer. Since it seems that there is no way to discern which launcher we are using, the only alternative that came to my mind is to add some meta-data here and use it to discern if we launched the tests in debug mode (which also means that we are using puppeteer)

Reply:
Nice, clean workaround 👍